src/figuresClasses.ts

```diff
@@ -1,19 +1,63 @@
+/* eslint-disable max-len */
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
 export class Triangle {
+  public shape = 'triangle';
+
+  constructor(public color: string, public a: number, public b: number, public c: number) {
```

Comment:
You should change this constructor to base on types from Figure interface.

Please try `constructor(
    public color: 'red' | 'green' | 'blue',
    public a: number,
    public b: number,
    public c: number,
    public shape: 'triangle' | 'circle' | 'rectangle',
  )`
  
  Then we are strictly sure that our values be always the same as we declared in types :)
  Try to fix other classes.